pyproject.toml

```diff
@@ -34,3 +34,8 @@ keywords = ["BCDI", "bragg", "diffraction", "imaging"]
 Repository = "https://github.com/advancedPhotonSource/cohere.git"
 Documentation = "https://cohere-dev.readthedocs.io/en/latest/"
 Issues = "https://github.com/advancedPhotonSource/cohere/issues"
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["cohere_core.*"]
```

Comment:
Only the core package is included in this project. The UI package should be built and published separately.